src/main/resources/mybatis/collections.xml

```diff
@@ -79,34 +79,34 @@
     <select id="getCollectionsJson" parameterType="map" resultType="String">
         select json_build_object(
             'collections', json_agg(collections),
-             'links' , json_build_array(
-                 json_build_object(
-                      'type', 'application/json',
-                       'rel',  'self',
-                       'title', 'This document as JSON',
-                       'href',  #{serverUrl} || 'collections?f=json'
-                 )
-             )
+            'links' , json_build_array(
+                json_build_object(
+                    'type', 'application/json',
+                    'rel',  'self',
+                    'title', 'This document as JSON',
+                    'href',  #{serverUrl} || 'collections?f=json'
+                )
+            )
         )
         from (
-            select <include refid="collectionColumnsToJson"/>
-                from (<include refid="selectCollectionsQuery"/>) as cols
+        select <include refid="collectionColumnsToJson"/>
+        from (<include refid="selectCollectionsQuery"/>) as cols
         ) as collections
     </select>
 
     <select id="getCollectionJson" parameterType="map" resultType="String">
-      select row_to_json(collection)
+        select row_to_json(collection)
         from (
-            select <include refid="collectionColumnsToJson"/>
-                from (<include refid="selectCollectionsQuery"/>) as cols
+        select <include refid="collectionColumnsToJson"/>
+        from (<include refid="selectCollectionsQuery"/>) as cols
         ) as collection
     </select>
 
 
     <select id="getStatisticalTimeSeriesJson" parameterType="java.util.Map" resultType="String">
         <!-- The aliases in this query are used for the JSON key names and some are intentionally mixed case -->
         <!-- The inner select distinct is necessary until we have a top level table with this information -->
-        select row_to_json(fsts)
+        <![CDATA[select row_to_json(fsts)
```

Comment:
To avoid having to add the<![ CDATA and the closing ]]> , the <= could be written as:
current_date - time_step between 0 and 1095